dataplane/src/main.rs

```diff
@@ -61,11 +61,12 @@ fn main() {
     /* parse cmd line args */
     let args = CmdArgs::parse();
 
-    let grpc_address = "[::1]:50051".parse().expect("Bad grpc address");
-
-    if let Err(e) = start_mgmt(grpc_address) {
+    // Get the gRPC address from command line args
+    let grpc_addr = args.get_grpc_address();
+    info!("Starting gRPC server on address: {}", grpc_addr);
```

Comment:
@sergeymatov I would remove this log, since the gRPC server is actually not starting at this point but later and there is already a log for that.